buildSrc/src/main/java/Dependencies.kt

```diff
@@ -20,7 +20,7 @@ object Versions {
     const val findbugs = "3.0.1"
     const val lottie = "2.5.5"
     const val leakcanary = "1.5.4"
-    const val telemetry = "0.5.1"
+    const val android_components = "0.10"
```

Comment:
Is there a particular reason why you are using 0.10 and not the latest release (0.16.1)?

Reply:
Will do!